File: picker/src/main/java/ir/shahabazimi/instagrampicker/gallery/GalleryModel.java
package ir.shahabazimi.instagrampicker.gallery;

public class GalleryModel {

    private String address;
    private boolean isSelected;

    GalleryModel(String address, boolean isSelected) {
        this.address = address;
        this.isSelected = isSelected;
    }

    String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    boolean isSelected() {
        return isSelected;
    }

    void setSelected(boolean selected) {
        isSelected = selected;
    }
}

File: picker/src/main/java/ir/shahabazimi/instagrampicker/models/GalleryModel.kt
package ir.shahabazimi.instagrampicker.models

import java.io.Serializable

/**
 * @Author: Shahab Azimi
 * @Date: 2023 - 03 - 22
 **/
data class GalleryModel(
    val address: String,
    var selectable: Boolean = false,
    var isSelected: Boolean = false
) : Serializable
